server/backend/backend.go

```diff
@@ -217,14 +217,14 @@ func (b *Backend) SendTransaction(ctx context.Context, tx sdk.Transaction) error
 		case *types.FlowError:
 			// TODO - confirm these
 			switch t.FlowError.(type) {
-			case *fvmerrors.AccountAuthorizationError,
```

Comment:
does the dereference happen implicitly here?

Reply:
No, it seems like the types have been changed from pointer to non-pointer error types.